.github/workflows/tests.yml

```diff
@@ -4,43 +4,46 @@ on:
   push:
     branches: [main]
   pull_request:
+  workflow_call:
```

Comment:
workflow_call: is for reusable workflows (calling from another workflow). You probably want workflow_dispatch: